src/tests/test_certstor.cpp

```diff
@@ -0,0 +1,257 @@
+/*
+* (C) 2016 Kai Michaelis, Rohde & Schwarz Cybersecurity
+*
+* Botan is released under the Simplified BSD License (see license.txt)
+*/
+
+#include "tests.h"
+
+#if defined(BOTAN_HAS_CERTSTOR_SQLITE3)
+   #include <botan/certstor_sqlite.h>
+   #include <botan/sqlite3.h>
+   #include <botan/internal/filesystem.h>
+   #include <botan/pkcs8.h>
+   #include <botan/auto_rng.h>
+   #include <sstream>
+   extern "C" {
+   #include <unistd.h> // unlink()
```

Comment:
This include for `unlink` will be trouble, not sure what to do here past adding an appropriate wrapper in filesystem.h and hoping C++17 gets here eventually.

(Also how is this even compiling on Windows/MinGW?)